modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -1345,13 +1345,19 @@ public ChecksumAndRevisions checksumAndRevisions(long revision) {
         }
     }
 
-    private long minChecksummedRevisionOrZero() {
+    private long minChecksummedRevisionOrZero() throws RocksDBException {
         try (
-                var options = new ReadOptions();
+                var options = new ReadOptions().setTailing(true);
                 RocksIterator it = revisionToChecksum.newIterator(options)
         ) {
             it.seekToFirst();
-            return it.isValid() ? bytesToLong(it.key()) : 0;
+
+            if (it.isValid()) {
+                it.status();
```

Comment:
This should be used the opposite way, actually) You need to check the status only if the iterator is not valid